core/src/main/java/de/gurkenlabs/litiengine/gui/GuiComponent.java

```diff
@@ -1013,7 +995,8 @@ public void setLocation(final double x, final double y) {
    * @param location the new location
    */
   public void setLocation(final Point2D location) {
-    this.setLocation(location.getX(), location.getY());
+    this.location = location;
+    this.boundingBox = null; // trigger recreation in next boundingBox getter call
```

Comment:
Child GUI components are not updated. setX() and setY() should be called, or children otherwise updated.